### ISSUE TITLE
E-Mail Templates durchsuchbar machen .. 

### ISSUE THREAD

**Issue by dergel** [yakamara/redaxo_yform#406]

**Comment by tbaddade** [comment]
auch via Watson?

**Comment by dergel** [comment]
wäre cool :)

**Comment by alexplusde** [comment]
@tbaddade sehe ich das richtig, man müsste das hier auf die E-Mail-Templates-Tabelle übertragen?

https://github.com/FriendsOfREDAXO/sked/tree/master/lib/watson

**Comment by tbaddade** [comment]
Du bist im falschen Repo … Das ist Sked
https://github.com/FriendsOfREDAXO/sked/tree/master/lib/watson

Die Anpassungen gehören momentan für YForm hier hin
https://github.com/tbaddade/redaxo_watson/tree/master/lib/Watson/Workflows/YForm

Dazu müsste es eine Class `YFormEmailTemplateSearch` erstellt werden und diese im Provider registriert werden.

**Comment by alexplusde** [comment]
> Du bist im falschen Repo … Das ist Sked

Nene, ich war schon richtig ;) Ich hätte mir das von Sked abgeschaut und hier in YForm eingebaut.

Aber jetzt sehe ich, dass Watson schon was für YForm mitbringt und damit ist mir auch klarer, dass mein Ansatz verkehrt ist.

Sollen wir dann das Issue hier zu machen und bei dir drüben auf?

**Comment by tbaddade** [comment]
> Sollen wir dann das Issue hier zu machen und bei dir drüben auf?

Nein, da der Workflow eigentlich zu YForm gehört. Watson sollte max. die Core-AddOns mitbringen. Das ist halt noch eine grundlegende Entscheidung. Will Watson YForm durchsuchen oder will YForm Watson nutzen um durchsucht zu werden und Watson selbst nur die API bereitstellt?

**Comment by alexplusde** [comment]
Ich habe einen Entwurf hierzu umgesetzt und bitte um Feedback. Wenn ich glaube, verstanden zu haben, wie das funktioniert, würde ich gerne noch weitere Addons an Watson anbinden. 💙 @tbaddade

**Comment by tbaddade** [comment]
Danke aber

> Das ist halt noch eine grundlegende Entscheidung.

Ich würde das gern erst einmal abwarten. Daraus folgt dann wo die Suche zum Schluss integriert wird.
